tests/test_message.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+# This is a hack for now until directory structure is sorted
+# When this is removed remember to un-ignore E402 in flake8
+import sys
+TEST_DIR = os.path.dirname(os.path.abspath(__file__))
+PROJECT_DIR = os.path.abspath(os.path.join(TEST_DIR, os.pardir))
+sys.path.insert(0, PROJECT_DIR)
```

Comment:
Are we using relative path because we are using __file__? I looked up on the internet and I haven't gotten a clear answer. I assume it worked when you tested.